src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;
+  padding: 1rem;
+  max-width: 20rem;

```

Comment:
Max-width en flex-basis doen hetzelfde wanneer je ze in een wrapper gebruikt die op row staat (`testimonial-section`) dus `max-width` mag weg :)